settings.gradle

```diff
@@ -41,3 +41,5 @@ file('modules').eachDir { dir ->
 
 include "docs-examples"
 project(":docs-examples").projectDir = file("docs/examples")
+include 'test-support'
```

Comment:
Will be published to Maven by default, let's disable it

Reply:
Yeah, I was intending to do that, and also for the docs-examples JAR.

Reply:
Just about to look into how best to do this.